### PR TITLE
Updated for 3 node issue

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated for 3 node issue

### CODE REVIEW

File: docs/Consensus-Protocols/Comparing-PoA.md [hunk 1]
```diff
@@ -20,24 +20,33 @@ and add blocks to the blockchain. Nodes are added to or removed from the signer/
...
 
 ### Immediate Finality 
 
 IBFT 2.0 has immediate finality. When using IBFT 2.0 there are no forks and all valid blocks are included in the main chain.
 
 Clique does not have immediate finality. Implementations using Clique must be aware of forks and chain reorganizations occurring. 
 
+### Minimum Number of Validators 
+
+IBFT 2.0 requires 4 validators to be Byzantine fault tolerant.
```
Comment (rain-on) [inline docs/Consensus-Protocols/Comparing-PoA.md]: Given this is the comparison file - should something be said about Clique's requirement? eg. Clique is able to operate with a single signer, however this offers no redundancy in the event of a node failure?

File: docs/Consensus-Protocols/Comparing-PoA.md [hunk 2]
```diff
@@ -20,24 +20,33 @@ and add blocks to the blockchain. Nodes are added to or removed from the signer/
...
+
+IBFT 2.0 requires 4 validators to be Byzantine fault tolerant. 
+
 ### Liveness 
 
 Clique is more fault tolerant than IBFT 2.0. Clique tolerates up to half to the validators failing. IBFT 2.0 networks 
-tolerate up to (n-1)/3 faulty nodes. For example, in an IBFT 2.0 network of:
+require greater than or equal to 2/3 of validators to be operating to create blocks. For example, in an IBFT 2.0 network of:
+
+* 4-5, 1 bad node is tolerated
```
Comment (rain-on) [inline docs/Consensus-Protocols/Comparing-PoA.md]: this is actually a bit funny - ratio of good vs bad nodes is a safety thing, not a liveness thing ..... so maybe rather than bad - this could be "unresponsive"?

Also - I think we'll find that 9 nodes supports 3 bad nodes (6-8 support 2 unresponse, 9-11 support 3 unresponive).

File: docs/Consensus-Protocols/Comparing-PoA.md [hunk 3]
```diff
@@ -20,24 +20,33 @@ and add blocks to the blockchain. Nodes are added to or removed from the signer/
...
-tolerate up to (n-1)/3 faulty nodes. For example, in an IBFT 2.0 network of:
+require greater than or equal to 2/3 of validators to be operating to create blocks. For example, in an IBFT 2.0 network of:
+
+* 4-5, 1 bad node is tolerated 
+* 6-9, 2 bad nodes are tolerated 
 
-* 3, no bad node are tolerated
-* 4-6, 1 bad node is tolerated 
-* 7-9, 2 bad nodes are tolerated 
+!!! important
```
Comment (rain-on) [inline docs/Consensus-Protocols/Comparing-PoA.md]: This should go under a safety heading, not liveness ... but ... meh?

File: docs/Consensus-Protocols/Comparing-PoA.md [hunk 4]
```diff
@@ -20,24 +20,33 @@ and add blocks to the blockchain. Nodes are added to or removed from the signer/
...
+* 6-9, 2 bad nodes are tolerated 
 
-* 3, no bad node are tolerated
-* 4-6, 1 bad node is tolerated 
-* 7-9, 2 bad nodes are tolerated 
+!!! important 
+    An IBFT 2.0 network with 3 validators can tolerate one bad node (that is, continue to produce blocks)
+    but is not Byzantine fault tolerant. 
+    
+    We do not recommend using IBFT 2.0 networks with 3 nodes for production purposes.
```
Comment (rain-on) [inline docs/Consensus-Protocols/Comparing-PoA.md]: .. with 3 or few nodes .... 
Also, do we "not recommend" or "We recommend not using IBFT ..." (i.e. are we not recommending, or are we actively recommending against?)

File: docs/Consensus-Protocols/Comparing-PoA.md (hunk 2 above)
Comment (Errorific) [reply docs/Consensus-Protocols/Comparing-PoA.md]: yeah, replace bad with unresponsive.

File: docs/Consensus-Protocols/Comparing-PoA.md [hunk 5]
```diff
@@ -20,24 +20,33 @@ and add blocks to the blockchain. Nodes are added to or removed from the signer/
...
+require greater than or equal to 2/3 of validators to be operating to create blocks. For example, in an IBFT 2.0 network of:
+
+* 4-5, 1 bad node is tolerated 
+* 6-9, 2 bad nodes are tolerated 
 
-* 3, no bad node are tolerated
-* 4-6, 1 bad node is tolerated 
-* 7-9, 2 bad nodes are tolerated 
+!!! important 
+    An IBFT 2.0 network with 3 validators can tolerate one bad node (that is, continue to produce blocks)
```
Comment (Errorific) [inline docs/Consensus-Protocols/Comparing-PoA.md]: I'd rephrase this to "can tolerate one unresponsive node (...) but cannot tolerate any malicious nodes". Or Maybe "byzantine" rather than "malicious"

File: docs/Consensus-Protocols/Comparing-PoA.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Consensus-Protocols/Comparing-PoA.md]: Good point. Fixed.

File: docs/Consensus-Protocols/Comparing-PoA.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Consensus-Protocols/Comparing-PoA.md]: Fixed

File: docs/Consensus-Protocols/Comparing-PoA.md (hunk 4 above)
Comment (MadelineMurray) [reply docs/Consensus-Protocols/Comparing-PoA.md]: Changed to 'We recommend not' - actively recommending against

File: docs/Consensus-Protocols/Comparing-PoA.md (hunk 5 above)
Comment (MadelineMurray) [reply docs/Consensus-Protocols/Comparing-PoA.md]: Updated

File: docs/Consensus-Protocols/Comparing-PoA.md [hunk 6]
```diff
@@ -20,24 +20,36 @@ and add blocks to the blockchain. Nodes are added to or removed from the signer/
...
 
-* 3, no bad node are tolerated
-* 4-6, 1 bad node is tolerated 
-* 7-9, 2 bad nodes are tolerated 
+* 4-5, 1 unresponsive node is tolerated 
+* 6-8, 2 unresponsive nodes are tolerated 
+
+!!! important 
+    An IBFT 2.0 network with 3 validators can tolerate one unresponsive node (that is, continue to produce blocks)
+    but cannot tolerate any Byzantine nodes (that is, forks can occur with one Byzantine node).
```
Comment (Errorific) [inline docs/Consensus-Protocols/Comparing-PoA.md]: Don't think byzantine here should be capitalised. Also uncertain if byzantine is the right term as an unresponsive node is byzantine. Malicious isn't the right term either as it should encompass nodes that are misbehaving from bugs etc. Maybe just misbehaving?

File: docs/Consensus-Protocols/IBFT.md [hunk 7]
```diff
@@ -11,6 +12,10 @@ Pantheon implements the IBFT 2.0 Proof-of-Authority (PoA) consensus protocol. IB
 In IBFT 2.0 networks, transactions and blocks are validated by approved accounts, known as validators. 
 Validators take turns to create the next block. Existing validators propose and vote to add or remove validators. 
 
+## Minimum Number of Validators 
+
+IBFT 2.0 requires 4 validators to be Byzantine fault tolerant.
```
Comment (Errorific) [inline docs/Consensus-Protocols/IBFT.md]: byzantine shouldn't be capitalised

File: docs/Tutorials/Create-IBFT-Network.md [hunk 8]
```diff
@@ -6,6 +11,9 @@ consensus protocol](../Consensus-Protocols/IBFT.md).
 !!!important
     An Ethereum private network created as described here is isolated but not protected or secure. 
     We recommend running the private network behind a properly configured firewall.
+    
+    This tuturial configures a private network using IBFT 2.0 for education purposes only. IBFT 2.0 requires 
+    4 validators to be Byzantine fault tolerant.
```
Comment (Errorific) [inline docs/Tutorials/Create-IBFT-Network.md]: capitalised byzantine again

File: docs/Consensus-Protocols/Comparing-PoA.md (hunk 6 above)
Comment (MadelineMurray) [reply docs/Consensus-Protocols/Comparing-PoA.md]: Networks with 3 or less validators are able to produce blocks but do not guarantee finality when operating in adversarial environments.
